gst_client/gst_client_symbolic_link.sh

```diff
@@ -4,4 +4,4 @@
 # $1: target
 # $2: link name
 rm -f ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
-ln -s ${DESTDIR}${MESON_INSTALL_PREFIX}/$1 ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
+ln -s ./$1 ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
```

Comment:
Why not: `ln -s --relative ${DESTDIR}${MESON_INSTALL_PREFIX}/$1 ${DESTDIR}${MESON_INSTALL_PREFIX}/$2` ?

Reply:
Updated as suggested.

Reply:
I believe `--relative` is, sadly, a GNU extension, and not available in the [POSIX specification](https://www.unix.com/man-page/posix/1p/ln/). As such, this would break builds in non-Linux boxes, such as Mac. I'd keep the original suggestion in favor of portability:
```
ln -s ${DESTDIR}${MESON_INSTALL_PREFIX}/$1 ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
```